docs/lib/storage/fragments/flutter/upload/no-upload-data.md

```diff
@@ -0,0 +1,5 @@
+<amplify-callout warning>
+
+**Note**: Amplify Flutter currently only supports uploading files via the `uploadFile` function.
+
+</amplify-callout>
```

Comment:
We also don't support `downloadData` yet but we havent called that out.

Maybe we should remove this section for now, or say we are working on adding it?